### PR TITLE
[PROTOCOL-81] Adding Docs In Contracts II

### PR DESCRIPTION
**Pull request by sypeer**
It includes docs for:

- LoansInterface and Implementations (Ether and Token)
- LoansBase
- LoanTermsConsensusInterface and Implementation (LoanTermConsensus)

In each contract:
- Added docs in external / public / internal functions, constructor and events. (using @ annotations).
- The contract has a header describing the purpose and @author develop@teller.finance

### CODE REVIEW

File: contracts/base/LoanTermsConsensus.sol [hunk 1]
```diff
@@ -22,11 +7,24 @@ import "./Consensus.sol";
...
 contract LoanTermsConsensus is Consensus, LoanTermsConsensusInterface {
+    /* Mappings */
     mapping(address => mapping(uint256 => ZeroCollateralCommon.AccruedLoanTerms)) public termSubmissions;
     mapping(address => mapping(uint256 => bool)) public requestNonceTaken;
 
+    /**
+        @notice Processes the loan request
+        @param ZeroCollateralCommon.LoanRequest request Struct of the protocol loan request
+        @param ZeroCollateralCommon.LoanResponses request List of structs of the protocol loan responses
+        @return uint256 Interest rate
```
Comment (salazarguille) [inline contracts/base/LoanTermsConsensus.sol:24]: It should be @return interestRate Interest rate and the same for the other return params

File: contracts/base/LoanTermsConsensus.sol [hunk 2]
```diff
@@ -71,6 +69,12 @@ contract LoanTermsConsensus is Consensus, LoanTermsConsensusInterface {
         );
     }
 
+    /**
+        @notice Processes the loan response
+        @param ZeroCollateralCommon.LoanRequest request Struct of the protocol loan request
```
Comment (salazarguille) [inline contracts/base/LoanTermsConsensus.sol:74]: It should be @param request struct of the protocol loan request

File: contracts/base/LoanTermsConsensus.sol (hunk 1 above)
Comment (sypeer) [reply contracts/base/LoanTermsConsensus.sol:24]: fixed

File: contracts/base/LoanTermsConsensus.sol (hunk 2 above)
Comment (sypeer) [reply contracts/base/LoanTermsConsensus.sol]: fixed

File: contracts/base/LoansBase.sol [hunk 3]
```diff
@@ -305,10 +330,23 @@ contract LoansBase is LoansInterface, Base {
     }
 
     /** Internal Functions */
-
+    /**
+        @notice Payouts the collateral for a loan
```
Comment (alex-murashkin) [inline contracts/base/LoansBase.sol:334]: I think it’s meant to be “pays out”

Comment (sypeer) [reply contracts/base/LoansBase.sol]: Fixed, thank you

**Pull request merged**